src/app/pages/manage-organisation/findyouradministrator/findyouradministrator.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Rename the file name and folder name to follow the current naming conventions 
ex:- forgot-password-success (include dashes between words)